src/NHibernate.Test/Associations/OneToOneFixture.cs

```diff
@@ -0,0 +1,167 @@
+using System;
+using System.Linq;
+using NHibernate.Cfg.MappingSchema;
+using NHibernate.Mapping.ByCode;
+using NUnit.Framework;
+
+namespace NHibernate.Test.Associations.OneToOne
```

Comment:
Namespace does not match the folder structure.

Reply:
Intention is to hide entities from `NHibernate.Test.Associations` namespace. Not sure what's best way to proceed. Seems separate folder is excessive here.

Reply:
You can make entities internal to the fixture class.

Reply:
Not sure what you meant. I can't put them inside class. As that breaks db creation as our schema creator  tries to create tables that look something like `OuterClass+InnerClass`. And `+` doesn't work with  SQL Server.

I think I will just wrap entities in separate namespace.